src/IO/H5/Header.cpp

```diff
@@ -18,19 +20,46 @@ Header::Header(const bool exists, detail::OpenGroup&& group,
                const hid_t location, const std::string& name)
     : group_(std::move(group)) {
   if (exists) {
-    header_info_ =
-        h5::read_rank1_attribute<std::string>(location, name + extension())[0];
+    header_info_ = h5::read_rank1_attribute<std::string>(location,
+                                                         name + extension())[0];
+    const auto printenv_location =
+        header_info_.find("############### printenv ###############\n") +
```

Comment:
Would it not be safer to put these repeated magic strings in some const variable to be reused? Safer in the sense that it reduces the likelihood of someone modifying just one of the repeated strings and breaking something...